docs/main/additional-info/alphafeatures.md

```diff
@@ -45,3 +46,162 @@ For additional details and detailed instructions on how to get started with Open
 OpenEBS is developing a monitoring add-on package that can be installed via helm or kubectl for setting up a default prometheus, grafana and alert manager stack. The package also will include default service monitors, dashboards and alert rules. 
 
 For additional details and detailed instructions on how to get started with OpenEBS Monitoring Add-on please refer this [Quickstart guide](https://github.com/openebs/monitoring).
+
+## Data Populator
+
+The Data populator can be used to load seed data into a Kubernetes persistent volume from another such volume. The data populator internally uses Rsync, which is a volume populator having the ability to create a volume from any rsync source. 
+
+### Use cases
+
+1. <b>Decommissioning of a node in the cluster</b>: In scenarios where a Kubernetes node needs to be decommissioned whether for upgrade or maintenance, a data populator can be used to migrate the data saved in the local storage of the node, that has to be decommissioned, to another node in the cluster. 
+2. <b>Loading seed data to Kubernetes volumes</b>: Data populator can be used to scale applications without using read-write many operation. The application can be pre-populated with the static content available in an existing PV.
+
+
+### Prerequisites 
+ 
+1. Kubernetes version should be >= 1.22
+2. Ensure `AnyVolumeDataSource` feature gate is enabled on the cluster.
+
+### Setting up the Data Populator
+
+1. To get started, the data populator CRD needs to be installed on the setup. To install, execute:
+
+ ```
+ kubectl apply -f https://raw.githubusercontent.com/openebs/data-populator/master/deploy/crds/openebs.io_datapopulators.yaml
+ ```
+2. Next, to install the data populator controller, execute:
+
+ ```
+ kubectl apply -f https://raw.githubusercontent.com/openebs/data-populator/master/deploy/yamls/data-populator.yaml
+ ```
```

Comment:
The two steps here can consolidated into one step. Checkout the latest doc here -> https://github.com/openebs/data-populator/blob/develop/docs/data-populator/data-populator.md